src/test/java/teammates/test/cases/datatransfer/FeedbackResponseAttributesTest.java

```diff
@@ -44,4 +48,126 @@ public void testDefaultTimestamp() {
+        Instant expectedUpdatedAt = Instant.now();
+
+        FeedbackResponseAttributes observedFeedbackResponseAttributes = FeedbackResponseAttributes.builder()
+                .withFeedbackResponseId(expectedId)
+                .withFeedbackSessionName(expectedSessionName)
+                .withCourseId(expectedCourseId)
+                .withFeedbackQuestionId(expectedQuestionId)
+                .withFeedbackQuestionType(expectedQuestionType)
+                .withGiver(expectedGiver)
+                .withGiverSection(expectedGiverSection)
+                .withRecipient(expectedRecipient)
+                .withRecipientSection(expectedRecipientSection)
+                .withResponseMetaData(expectedResponseMetaData)
+                .withCreatedAt(expectedCreatedAt)
+                .withUpdatedAt(expectedUpdatedAt)
+                .build();
+
+        assertEquals(expectedId, observedFeedbackResponseAttributes.getId());
+        assertEquals(expectedSessionName, observedFeedbackResponseAttributes.feedbackSessionName);
+        assertEquals(expectedCourseId, observedFeedbackResponseAttributes.courseId);
+        assertEquals(expectedQuestionId, observedFeedbackResponseAttributes.feedbackQuestionId);
+        assertEquals(expectedQuestionType, observedFeedbackResponseAttributes.feedbackQuestionType);
+        assertEquals(expectedGiver, observedFeedbackResponseAttributes.giver);
+        assertEquals(expectedRecipient, observedFeedbackResponseAttributes.recipient);
+        assertEquals(expectedGiverSection, observedFeedbackResponseAttributes.giverSection);
+        assertEquals(expectedRecipientSection, observedFeedbackResponseAttributes.recipientSection);
+        assertEquals(expectedResponseMetaData, observedFeedbackResponseAttributes.responseMetaData);
+        assertEquals(expectedCreatedAt, observedFeedbackResponseAttributes.getCreatedAt());
+        assertEquals(expectedUpdatedAt, observedFeedbackResponseAttributes.getUpdatedAt());
+    }
+
+    @Test
+    public void testBuilderCopy() {
+        String originalGiver = "giver";
+        String originalRecipient = "recipient";
+        String originalQuestionId = "someQuestionId";
+        String originalId = originalQuestionId + "%" + originalGiver + "%" + originalRecipient;
+        FeedbackResponseAttributes original = FeedbackResponseAttributes.builder()
+                .withFeedbackResponseId(originalId)
+                .withFeedbackResponseId("originalFeedbackQuestionId%originalGiver%")
+                .withFeedbackSessionName("originalName")
+                .withCourseId("originalCourseId")
+                .withFeedbackQuestionId(originalQuestionId)
+                .withFeedbackQuestionType(FeedbackQuestionType.TEXT)
+                .withGiver(originalGiver)
+                .withRecipient(originalRecipient)
+                .withResponseMetaData(new Text("original meta data"))
+                .withCreatedAt(Instant.now())
```

Comment:
Following the discussion from https://github.com/TEAMMATES/teammates/pull/8741#discussion_r178560481, I think the intuition behind it is a valid one and has to do with the determinism of tests.

Because we are unit testing copying, no other things should affect the test:
* What happens if `Instant.now()` has side effects? For example it mutates some global state that can cause test failures, or it cannot be initialized for some reason.

This is nearly impossible here because the abstraction of `Instant` is great but the idea is that we are depending upon classes state already why should we even take into account that their states can randomly change over time?

In this case, `Instant.now()` having a different state over time has no impact because it's just a point in time.

* Usually tests have well defined inputs and outputs, but in this test the inputs and outputs change across time. The tests that we want to have inputs and outputs changing are usually for random testing, and we should make clear it is a random test.

With that said this is a small issue so up to you to decide :). The inconsistency is glaring though (why are all the fields constant and then all of a sudden we have a non-constant field?).